lib/elasticsearch-cache.js

```diff
@@ -0,0 +1,206 @@
+const _ = require('lodash');
+const colors = require('colors');
+const moment = require('moment');
+const client = require('./elasticsearch')();
```

Comment:
@fqjony I would recommend to initialize client in the constructor. And only if client is not provided in arguments.